oas_docs/overlays/kibana.overlays.yaml

```diff
@@ -14,3 +14,42 @@ actions:
         variables:
           kibana_url:
             default: localhost:5601
+  - target: '$.components.securitySchemes'
+    description: Add an API key security scheme
+    update:
+      apiKeyAuth:
+        in: header
+        name: Authorization
+        type: apiKey
+        description: You must create an API key and use the encoded value in the request header. To learn about creating keys, go to [API keys](https://www.elastic.co/guide/en/kibana/current/api-keys.html).
+  - target: '$.components.securitySchemes'
+    description: Add an basic security scheme
+    update:
+      basicAuth:
+        type: http
+        scheme: basic
+  - target: '$'
+    description: Add document-level security.
+    update:
+      security:
+        - apiKeyAuth: []
+        - basicAuth: []
+  - target: '$'
+    description: Add an extra page about spaces
+    update:
+      x-topics:
```

Comment:
I'm not familiar with `x-topics` but this sounds like a good introduction to the functionality of spaces!

IMO it would be cool if we had a section "spaces" in our OAS that also contained the `/s/{space_id}/{*path}` with a description of the params but this approach seems like a nice simple starting point!